src/pip/_internal/locations.py

```diff
@@ -72,19 +72,27 @@ def get_src_prefix():
     # Use getusersitepackages if this is present, as it ensures that the
     # value is initialised properly.
     user_site = site.getusersitepackages()
+    user_base = site.getuserbase()
 except AttributeError:
     user_site = site.USER_SITE
+    user_base = site.USER_BASE
 
 if WINDOWS:
     bin_py = os.path.join(sys.prefix, 'Scripts')
-    bin_user = os.path.join(user_site, 'Scripts')
+    if running_under_virtualenv():
+        bin_user = os.path.join(user_site, 'Scripts')
+    else:
+        bin_user = os.path.join(user_base, 'Scripts')
     # buildout uses 'bin' on Windows too?
     if not os.path.exists(bin_py):
         bin_py = os.path.join(sys.prefix, 'bin')
         bin_user = os.path.join(user_site, 'bin')
 else:
     bin_py = os.path.join(sys.prefix, 'bin')
-    bin_user = os.path.join(user_site, 'bin')
+    if running_under_virtualenv():
+        bin_user = os.path.join(user_site, 'bin')
```

Comment:
That's still not it. 
```
>>> import site
>>> from pip._internal.utils.virtualenv import running_under_virtualenv
>>> running_under_virtualenv()
True
>>> user_site = site.USER_SITE
>>> bin_user = os.path.join(user_site, 'bin')
>>> bin_user
'/home/user/.local/lib/python3.8/site-packages/bin'
```

This is extremely hacky und ugly, but finds the right path
```python
os.path.dirname(os.path.dirname(os.path.dirname(site.getsitepackages()[0])))
```
evaluating to `'/home/user/testenv'`